mitodl/mitxpro
# feat: add products and product versions for emeritus products

### What are the relevant tickets?




A part of https://github.com/mitodl/hq/issues/4032

### Description (What does it do?)

This PR adds extra functionality to the Emeritus API ingestion to create products and product versions.


### How can this be tested?


- On master, run `./manage.py sync_external_course_runs --vendor emeritus`
- Notice that you do not have any product or product versions for the Emeritus course runs.
- Now create a few products and product version for a few emeritus course runs
- Create a couple of products with correct prices and a couple with incorrect prices.
- Now checkout this branch and run the above command.
- It should create products and product versions for any runs with products and versions.
- It should not create products or versions for the correct products and versions created above.
- It should create a new version for the incorrect prices created above.
